testsuite/meta/kernel/sigmoid.dat

```diff
@@ -1 +1 @@
999999970907,0.9999971136530981,0.999999835032178,0.9999998281309567,0.9999999739939959,0.9993710419929885,0.999999999751557],[-0.9837088229160907,-0.9576239368138867,-0.7154088181950056,-0.9876047314223816,-0.9806374782649956,-0.9041804741756538,-0.9536405398438321,-0.9040887965020897,-0.9795112814305048,-0.3290913135381032,-0.9952387912872487,-0.9718288802793585,-0.911123591559726,-0.8399383995647365,-0.9585072603667272,-0.9212955463478065,-0.986919868823827,-0.8319264870214883,-0.3532103112230849,-0.8099730165385597,-0.8255550380052745,-0.7217307356789098,-0.5848658833175268,0.9973659273022059,0.9997151717688844,0.9991327694932861,0.9988044505215239,0.9999779780914132,0.9999608446024446,0.9997849964261339,0.9989197118498461,0.990414640228012,0.999832036054119,0.9997505489688681,0.9978120084401704,0.998374738289247,0.9987928041289634,0.9995137131294393,0.9924963601547339,0.9998701475179471,0.9974821927610142,0.99975524053716,0.9992688744855665,0.9996052582164795,0.9962393871302818,0.9999450948460912,0.9943064299734047,0.9949527290866435,0.9925774295522892,0.8431359442941078,0.9938178365456201,0.9749468297003453,0.9998168974123098,0.9748215186627072,0.9974940481875011,0.9930174811660948,0.9982628423666333,0.9265806190033976,0.992880218474236,0.998643749881832,0.9978029908999805,0.9820041352918771,0.9607616387563029,0.8779665940402657,0.990535966397355,0.9897603000038034,0.9753907326617822,0.9970844556257493,0.9728439634087293,0.994172023317521,0.9872439815224494,0.7735435605918188,0.9835509623644464,0.9885445716480773,0.9979257629251014,0.8622949524031028,0.9452476474755828,0.9900515059604838,0.9967231333358247,0.9862045689398787,0.9986670000807658,0.9767769722729528,0.9775631038636651,0.8637194070454496,0.9611837898970099,0.9997213468931843,0.9857620542710723,0.9937024903150054,0.990027997507986,0.9965454776647982,0.9641973620743289,0.999538196249901],[-0.999999999898996,-1.0,-0.99999999466595,-0.9999999999999995,-0.999999999999987,-0.9999999999999188,-0.9999999999105278,-0.9999999997536251,-0.9999999999980446,-0.999996226810638,-1.0,-0.9999999999997408,-0.999999999999796,-0.9999999999987495,-0.9999999999063406,-0.9999999189773065,-0.999999999999999,-0.999999992946592,-0.9965752517667231,-0.999999999996715,-0.999562304092296,-0.9999998791079869,-0.9997778215352635,0.9999999821188984,1.0,0.9999999993536519,0.9999999994158619,1.0,1.0,1.0,0.9999999999867432,0.9999911862039167,1.0,1.0,0.9999999995356556,0.9999999383182878,0.9999999996177107,0.9999999999953143,0.9999949721134184,1.0,0.9999999953587726,0.9999999999941689,1.0,0.9999999983908846,0.999999989271515,1.0,0.9999999947159914,0.9999999999999879,0.9999999832173728,0.9999999447938291,0.9999999997415168,0.9996346036924734,1.0,0.9999999997493894,0.9999999991948243,0.9999967819124732,0.9999999999990167,0.9999998836320871,0.9999999999948191,1.0,0.9999999999999999,0.9999986250601111,0.9999999999999963,0.9999998064918568,0.9999999999999962,0.9999999999817117,0.9999999999999738,0.9999999999999666,0.99
```

Comment:
Note that these values changed because there was a bug in auto initialisation, where the user provided parameter values were always being overridden by the auto initialiser